calendar-amqp/src/test/java/com/linagora/calendar/amqp/EventIndexerConsumerTest.java

```diff
@@ -517,7 +517,7 @@ void shouldUpdateRecurrenceEvent() {
             ORGANIZER;CN=John1 Doe1:mailto:{organizer}
             ATTENDEE;PARTSTAT=NEEDS-ACTION;RSVP=TRUE;ROLE=REQ-PARTICIPANT;CUTYPE=INDIVI
              DUAL;CN=John2 Doe2;SCHEDULE-STATUS=1.1:mailto:{attendee1}
-            ATTENDEE;PARTSTAT=ACCEPTED;RSVP=FALSE;ROLE=CHAIR;CUTYPE=INDIVIDUAL:{organizer}
+            ATTENDEE;PARTSTAT=ACCEPTED;RSVP=FALSE;ROLE=CHAIR;CUTYPE=INDIVIDUAL:mailto:{organizer}
```

Comment:
Please have at least one test to ensure we support the old syntax without mailto

Reply:
I want no regression in behaviours.
Please add the tet I asked for.